ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/SRMA/Index.cshtml.cs

```diff
@@ -1,44 +1,58 @@
 ﻿using ConcernsCaseWork.Pages.Base;
+using ConcernsCaseWork.Services.Cases;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using System;
+using System.Linq;
 using System.Threading.Tasks;
+using CaseActionModels = ConcernsCaseWork.Models.CaseActions;
 
-namespace ConcernsCaseWork.Pages.Case.Management.Action.Srma
+namespace ConcernsCaseWork.Pages.Case.Management.Action.SRMA
 {
 	[Authorize]
 	[ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
 	public class IndexPageModel : AbstractPageModel
 	{
+		private readonly ISRMAService _srmaService;
 		private readonly ILogger<IndexPageModel> _logger;
 
-		public IndexPageModel(ILogger<IndexPageModel> logger)
+		public long CaseUrn { get; set; }
+		public CaseActionModels.SRMA SRMA { get; set; }
+
+		public IndexPageModel(ISRMAService srmaService, ILogger<IndexPageModel> logger)
 		{
+			_srmaService = srmaService;
 			_logger = logger;
 		}
 
 		public async Task OnGetAsync()
 		{
-			try
+			if (long.TryParse(Convert.ToString(RouteData.Values["urn"]), out long caseUrn) == false || caseUrn <= 0)
```

Comment:
Can we make this check similar to this `(caseUrnValue == null || !long.TryParse(caseUrnValue.ToString(), out long caseUrn) || caseUrn == 0)` for the sake of consistency